new-pipeline/src/evo.js

```diff
@@ -758,11 +757,7 @@ function displayEvolution(target, lines, usePercentages, plotOptions) {
     y_label: "",
     transition_on_update: false,
     interpolate: "linear",
-    yax_format: usePercentages ? function (y) {
-      return y + "%";
-    } : function (y) {
-      return y;
-    },
+    yax_format: (y => y + valueSuffux),
```

Comment:
This coerces y to a string which, for counts like submissions and metric counts, means the formatter can't automatically translate the y-ticks from, say, 150000 to 150k.